e2e/.env.sample

```diff
@@ -4,5 +4,6 @@ WRITER_USER=??
 READER_USER=??
 COLLABORATOR_USER=??
 ACCESS_TEST_USER=??
+ADMIN_TEST_USER=??
```

Comment:
Note: I am pretty lazy about creating local accounts, especially as each one requires upkeep. How many of these are necessary to run the standard puppeteer test suite, **and** how many of these can overlap. FOr the existing ones, that may be an @aweng98  question. It may be worth updating the README to explain how many users someone is expected to create locally.

For example: can `ADMIN_TEST_USER == USER_NAME` locally?

Reply:
@chavanr2019 ^